### PR TITLE
[Workers] Make --from-dash more apparent in dashboard starters

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/workers/_partials/_dash-creation-next-steps.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+2. Select **Workers & Pages**.
+3. Select your application.
+4. Select **Edit Code**.
+
+### Wrangler CLI
+
+To develop more advanced applications or [implement tests](/workers/testing/), start working in the Wrangler CLI:
+
+1. [Install Wrangler](/workers/wrangler/install-and-update/).
+2. Run `wrangler initi --from-dash {worker_name}`.
```
Comment (dario-piotrowicz) [inline content/workers/_partials/_dash-creation-next-steps.md]:
```suggestion
2. Run `wrangler init --from-dash {worker_name}`.
```

Comment (kodster28) [reply content/workers/_partials/_dash-creation-next-steps.md]: whoops! Thanks!

File: content/workers-ai/get-started/dashboard.md [hunk 2]
```diff
@@ -14,12 +14,17 @@ Follow this guide to create a Workers AI application using the Cloudflare dashbo
...
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
-2. Select **Workers & Pages** > **Create application**.
-3. Under **Create using a template**, select **LLM App**. After you select your template, an AI binding will be created for you in the dashboard.
-4. Review the provided code and select **Deploy**.
-5. Preview your Worker at its provided [`workers.dev`](/workers/configuration/routing/workers-dev/) subdomain.
\ No newline at end of file
+2. Go to **Workers & Pages**.
+3. Select **Create application**.
+4. Under **Create using a template**, select **LLM App**. After you select your template, an [AI binding] will be created for you in the dashboard.
```
Comment (Maddy-Cloudflare) [inline content/workers-ai/get-started/dashboard.md]:
```suggestion
4. Under **Create using a template**, select **LLM App**. After you select your template, an [AI binding] will be created for you in the dashboard.
```

Comment (Maddy-Cloudflare) [reply content/workers-ai/get-started/dashboard.md]: Did you want to add a link on `AI binding`?

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/workers-ai/get-started/dashboard.md [hunk 3]
```diff
@@ -14,12 +14,17 @@ Follow this guide to create a Workers AI application using the Cloudflare dashbo
...
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
-2. Select **Workers & Pages** > **Create application**.
-3. Under **Create using a template**, select **LLM App**. After you select your template, an AI binding will be created for you in the dashboard.
-4. Review the provided code and select **Deploy**.
-5. Preview your Worker at its provided [`workers.dev`](/workers/configuration/routing/workers-dev/) subdomain.
\ No newline at end of file
+2. Go to **Workers & Pages**.
+3. Select **Create application**.
+4. Under **Create using a template**, select **LLM App**. After you select your template, an [AI binding](workers-ai/configuration/bindings/) will be created for you in the dashboard.
```
Comment (kodster28) [inline content/workers-ai/get-started/dashboard.md]:
```suggestion
4. Under **Create using a template**, select **LLM App**. After you select your template, an [AI binding](/workers-ai/configuration/bindings/) will be created for you in the dashboard.
```

File: content/workers/_partials/_dash-creation-next-steps.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  {{<Aside type="note" header="Node.js version manager">}}
+  Use a Node version manager like [Volta](https://volta.sh/) or [nvm](https://github.com/nvm-sh/nvm) to avoid permission issues and change Node.js versions. [Wrangler](/workers/wrangler/install-and-update/), discussed later in this guide, requires a Node version of `16.17.0` or later.
+  {{</Aside>}}
+
+3. Run the following command, replacing the value of `[<DIRECTORY>]` which the location you want to put your Worker Script.
+
+  {{<tabs labels="npm | yarn | pnpm | bun">}}
+  {{<tab label="npm" default="true">}}
+  ```sh
+  $ npm create cloudflare [<DIRECTORY>] --type=pre-existing
```
Comment (dario-piotrowicz) [inline content/workers/_partials/_dash-creation-next-steps.md]: we're missing a necessary `--` here:
```suggestion
  $ npm create cloudflare [<DIRECTORY>] -- --type=pre-existing
```
(Note: it's just `npm`, the other pms don't need it)

**Pull request merged**